docs/tutorials/word2vec.ipynb

```diff
@@ -37,20 +37,20 @@
         "id": "AOpGoE2T-YXS"
```

Comment:
<p>"can to sample random words" -&gt; "can sample random words"</p><p> </p><p>Looks like there might be an extra line break between "You can" and "pass words from the positive class"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/text/pull/1229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Thanks for catching these, will fix.</p>